scripts/Tools/case.build

```diff
@@ -54,11 +54,31 @@ OR
     parser.add_argument("-m", "--model-only", action="store_true",
                         help="Assume shared libs already built")
 
+    comps = ("atm", "ice", "glc", "lnd", "ocn", "rof", "wav", "cpl")
```

Comment:
What if, instead of hardcoding the list of comps, you build it based on the subdirectories of EXEROOT?